test/testcafe/spec/SessionStorage_spec.js

```diff
@@ -369,5 +420,15 @@ test.requestHooks(identifyChallengeMock)('shall back to sign-in and authenticate
       accessToken: xhrSuccessTokens.access_token,
       idToken: xhrSuccessTokens.id_token
     }
-  ]);
+  ];
+  if (userVariables.v3) {
+    expectedMessages.push(...[
+      'afterRender',
+      {
+        controller: null,
+        formName: 'cancel'
+      },
+    ]);
+  }
```

Comment:
We can skip triggering `afterRender ` event if `idxTransaction.status === 'SUCCESS'` for the sake of parity with v2
https://github.com/okta/okta-signin-widget/blob/d207dcae5e21afb470dae0dca93e5c3881b2ee76/src/v3/src/components/Widget/index.tsx#L293

Reply:
Ahh cool! I didnt know v2 skipped it when it hits success status 🙏 

We can always do that as part of a separate ticket, so no need to now (want to avoid increasing the scope of this ticket). 

Ty!

Reply:
Created OKTA-604105